### PR TITLE
Adds a small known issue to the 4.9 RNs

### PR DESCRIPTION
**Pull request by stevsmit**
No BZ. Request from Lalatendu.

Preview: https://deploy-preview-39722--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes#ocp-4-9-9

For 4.9.

Can be merged upon approval.

### CODE REVIEW

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

**Comment by LalatenduMohanty** [comment]
> No BZ. Request from Lalatendu.

https://bugzilla.redhat.com/show_bug.cgi?id=2022570 this is the bug which got fixed in 4.9.9

**Review by wking — APPROVED** [review]

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2337,6 +2337,11 @@ link:https://access.redhat.com/solutions/6542521[{product-title} 4.9.9 container
 
 This update contains changes from Kubernetes 1.22.3. More information can be found in the following changelog: link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1223[1.22.3].
 
+[id="ocp-4-9-9-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Cluster Version Operator (CVO) ignored the `spec.overrides[].group` when deciding whether to override a manifest. Consequently, overridden entries might match multiple resources, which could override more resources than an admin might have intended. Additionally, overridden entries with an invalid group were considered a match, and admins might have been using invalid group values without noticing. With this update, the CVO requires group matching when applying configured overrides. As a result, the CVO no longer matches multiple manifests with a single override. Instead, the CVO only matches the manifest with the correct group. Admins who had been previously using an invalid group will have to update to the correct group in order to have their overrides continue to match. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2022570[*BZ#2022570*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-9-release-notes.adoc:2343]: s/admin/administrator (probably specifically "cluster administrator" but you might want Trevor or Lala to confirm that) throughout.

**Review by jeana-redhat — COMMENTED** [review]
Just one nit, LGTM

**Pull request merged**